### PR TITLE
Javascript javascript1 week4

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: javascript/javascript1/week4/voice-assistant.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+function getReply(command) {
+  // for (let i = 0; i < myName.length; i++)
+  if (command.includes("Hello my name is")) {
+    const newArr = command.split(" ");
+    myName.push(newArr[newArr.length - 1]);
+    console.log(myName);
+    return `Nice to meet you ${myName[myName.length - 1]}`;
+  }
+
+  // ===>>> This part of the code works, but doesn`t allow the function to execute the code further
```
Comment (Vahablotfi) [inline javascript/javascript1/week4/voice-assistant.js:12]: which part ? line 13-14? or rest of the code?

**Review by Vahablotfi — APPROVED** [review]
Everything looks good, well done in total. I only have two recommendations. 
1. You could use `switch()` to avoid a lot of `if ... else`.
2. You could use higher order functions, instead of for loops.

File: javascript/javascript1/week4/voice-assistant.js (hunk 1 above)
Comment (na-talia) [reply javascript/javascript1/week4/voice-assistant.js:12]: Line 13-14 (together with line 4)

Comment (Vahablotfi) [reply javascript/javascript1/week4/voice-assistant.js:12]: It could be because that `myName` is empty when you start for loop.

**Pull request merged**